src/main/java/Commands/ComplexCommand.java

```diff
@@ -0,0 +1,26 @@
+package Commands;
+
+import java.util.ArrayList;
+
+public class ComplexCommand {
+    ArrayList<DriverCommand> commands;
```

Comment:
Should be private.
List type (attribute and in the constructor) would be sufficient and would not limit the implementation.